src/cljs/gc_api_browser/response.cljs

```diff
@@ -20,17 +20,20 @@
 
 (defn render-body [body]
   (let [json-string (.stringify js/JSON (clj->js body) nil 2)]
-    (dom/pre #js {:style #js {:width "50%"
-                              :float "left"}}
+    (dom/pre #js {:style #js {:width "50%" :float "left"}}
              (dom/code nil json-string))))
 
-(defn component [{:keys [body headers status] :as resp} owner]
+(defn component [cursor owner]
   (reify
     om/IRender
     (render [_]
-      (let [json-string (.stringify js/JSON (clj->js body) nil 2)]
-        (dom/div #js {:className "well response"}
-                 (dom/h1 #js {:className "response-status"} status)
```

Comment:
@maxmurdoch we've lost the status code here. Where in the new design can that fit in?